code/modules/mob/living/carbon/xenomorph/life.dm

```diff
@@ -120,7 +121,17 @@
 	adjustBruteLoss(-amount)
 	adjustFireLoss(-amount)
 
+#define XENO_OVERHEALTH_PER_WARDING_LEVEL 25
```

Comment:
```suggestion
#define XENO_OVERHEALTH_PER_WARDING_LEVEL 20
```
Let's be a little more conservative here. From 20 to 100 extra raw HP seems like a lot already.

Reply:
given they have to build it up over time it feels fine, esp with the loss of all the damage reductions.

Reply:
Small health pool that regenerates fast is better than big health pool that takes a long time regenerating.
The waiting game is not very engaging in general, nor is facing a foe that has waited a long time.